src/components/NavMenu.tsx

```diff
@@ -72,23 +72,23 @@ const MenuItem = styled.li`
 const HomepageLink = styled(NavLink)<ColorProps>`
   ${color};
   &:hover {
-    color: white;
+    color: ${theme.colors.athensGray};
```

Comment:
lets use `copyOne` here and below too

Reply:
good shout

Reply:
I've replaced all instances of `${theme.colors.athensGray}` and `${theme.colors.black}` with `${theme.colors.copyOne}` and `${theme.colors.copyTwo}` respectively